colibris/app.py

```diff
@@ -103,11 +100,10 @@ def _init_routes(web_app):
 # APISpec/Swagger support
 
 def _init_swagger(web_app):
-    async def init_wrapper(app):
-        setup_swagger(app=app, swagger_url=settings.API_DOCS_PATH, swagger_info=app['swagger_dict'])
+    from colibris.docs.openapi.setup import setup_openapi_ui
```

Comment:
@ccrisan, do you see a problem with importing this setup here?

Reply:
Is this to avoid some circular import?

Reply:
No. To have the correct values for settings. Check `colibris/docs/openapi/__init__.py`.